examples/plot_mdm_spd_helper.py

```diff
@@ -0,0 +1,145 @@
+import numpy
+import matplotlib
+import matplotlib.pyplot as plt
+import matplotlib.gridspec as gridspec
+
+plt.style.use('seaborn')
+
+fontsize=15
+matplotlib.rc('font', size=fontsize)
+matplotlib.rc('text', usetex=True)
+matplotlib.rc('legend', fontsize=fontsize)
+matplotlib.rc('font',
+              family='times',
+              serif=['Computer Modern Roman'],
+              monospace=['Computer Modern Typewriter'])
+
+EPS=1e-8
+
+class DatasetSPD_2D():
+
+    def __init__(self,n_samples,n_features,n_classes):
+        self.n_samples=n_samples
+        self.n_features=n_features
+        self.n_classes=n_classes
+        self.data_helper=DataHelper()
+
+    def generate_sample_dataset(self):
+        X,Y=self.easy_gauss()
+        X,Y=self.data_helper.shuffle(X,Y)
+        return X,Y
+
+    def mat2vec(self,mat):
```

Comment:
Use the method `to_vector` from the class `SymmetricMatrices`?

Reply:
I won't be using it at all in fact; I've since then learned from @nguigs that SPD matrices will only work with `point_type=='matrix'` .